views/index.hbs

```diff
@@ -36,7 +36,9 @@
         {{#if prs}}
             <div class="row">
                 <div class="large-12 columns">
-                    <h2><strong>{{ prs.length }} / 4</strong> pull requests submitted in Hacktober</h2>
+                    <h2><strong>{{ prs.length }} / 4</strong> pull requests submitted in Hacktober.</h2>
+                    <br>
+                    <h3 class="center"><strong>{{ statement }}</strong></h3>
```

Comment:
Can you also add this to the line below line 69 as well (I know the logic here isn't perfect, but from what I can tell you can't do much in terms of expressions in handlebars and didn't bother adding a custom helper or anything)